docs/en/operations/settings/settings.md

```diff
@@ -2459,4 +2458,19 @@ Possible values:
 
 Default value: `0`.
 
+## execute_merges_on_single_replica_time_threshold {#execute-merges-on-single-replica-time-threshold}
+
+Enables special logic to perform merges on replicas.
+
+Possible values:
+
+-   Positive integer (in seconds).
+-   0 — Special merges logic is not executed.
+
+Default value: `0`.
+
+**Special logic to perform merges**
+
```

Comment:
Can be useful when merges are CPU bounded not IO bounded (heavy compression is in use, calculation expensive aggregate functions or default expressions or just very high number of tiny merges)